package.json

```diff
@@ -35,7 +35,8 @@
     "@babel/preset-typescript": "^7.10.4",
     "@braze/web-sdk-core": "3.1.0",
     "@emotion/babel-preset-css-prop": "^10.0.27",
-    "@emotion/core": "10.0.x",
```

Comment:
Note to my future self: check what effect this has on the Epic story where we use the window to share dependencies with the dynamically imported component. I suspect this may have broken things.

Reply:
This seems to work okay, we're still providing a dependency called `emotionCore` to the dynamically imported epic component, but it's coming from `@emotion/react` now.